api/app/utils/helpers.py

```diff
@@ -0,0 +1,34 @@
+""" API related helpers """
+from typing import List, Union
+import re
+
+
+
+def to_camel_case(s: str) -> str:
+    """
+    Converts a snake_case string to camelCase.
+
+    Args:
+        s: The snake_case string to convert.
+
+    Returns:
+        The resulting camelCase string.
+    """
+    s = re.sub(r"_(?!_)([a-z])", lambda m: m.group(1).upper(), s)
+    s = re.sub(r"_(?=_)", "", s)  # Remove consecutive underscores
+    return s
```

Comment:
what's the difference between this https://github.com/pizza-society/unicornify/blob/development/api/app/core/schemas/camel_case.py